js/src/components/campaign-conversion-notice/index.scss

```diff
@@ -0,0 +1,33 @@
+.gla-dashboard {
+    .gla-dashboard__campaign_conversion {
+		gap: var(--main-gap);
+		margin-bottom: var(--main-gap);
+
+		.gla-campaign-conversion-status-notice{
+			margin-left: 0px;
+			margin-right: 0px;
+			padding: $grid-unit-30 $grid-unit-40;
+
+			.components-notice__dismiss{
+				min-width: 1.85rem;
+				width: 1.85rem;
+				height: 1.85rem;
+			}
+
+			.gla-campaign-conversion-status-notice__title {
+				margin-bottom: $grid-unit-10;
+			}
+			
+			.gla-campaign-conversion-status-notice__external_icon{
+				margin-left: 0.2em;
+				vertical-align: middle;		
+				fill: currentColor;
+			}
+
+			.gla-campaign-conversion-status-notice__external_link{
+				margin-bottom: 0px;
+			}
```

Comment:
💅 
The link text `Learn more about this upgrade` should be stronger according to the design. Also the underline of the text should be omitted.

**Current style**
<img width="1030" alt="Screenshot 2022-05-04 at 15 21 18" src="https://user-images.githubusercontent.com/914406/166637809-a0fd779c-554f-4d38-86de-75fe5a816522.png">
**Design**
<img width="1209" alt="Screenshot 2022-05-04 at 15 21 44" src="https://user-images.githubusercontent.com/914406/166637852-89b118b8-9aaf-4706-b6b0-d0c518e5e381.png">

Reply:
Added here [359d5ff](https://github.com/woocommerce/google-listings-and-ads/pull/1472/commits/359d5ffcf4e83cc661f072a7edcb45334d728311)